src/angular/sourceMappingVisitor.ts

```diff
@@ -95,7 +95,7 @@ export class SourceMappingVisitor extends RuleWalker {
   parentAST: any;
   private consumer: SourceMapConsumer;
 
-  constructor(sourceFile: ts.SourceFile, options: IOptions, protected codeWithMap: CodeWithSourceMap, protected basePosition: number) {
+  constructor(sourceFile: ts.SourceFile, options: IOptions, public codeWithMap: CodeWithSourceMap, protected basePosition: number) {
```

Comment:
Why not `protected`?

Reply:
It was necessary because I've changed the type of `context` in `trackByRule#35` from `any` to `BasicTemplateAstVisitor` (https://github.com/mgechev/codelyzer/pull/659/files#diff-26d9296d5e0a25d3fe6a9b174e67da35R35)... it gives an error with `protected`:

> src/trackByFunctionRule.ts(44,30): error TS2446: Property 'codeWithMap' is protected and only accessible through an instance of class 'TrackByFunctionTemplateVisitor'.